### ISSUE TITLE
Use of disappearing popups (toasts) for upload/compile errors is inaccessible to screen readers

### ISSUE THREAD

**Issue by tigoe** [arduino/arduino-ide#166]
WIth a screen reader on, pop-up errors like the one below are not accessible because it disappears too soon:

```
Error: Request upload failed with message: 2 UNKNOWN: no upload port provided
```

Steps to reproduce:
1. Turn VoiceOver on on MacOS (commnad-F5)
2. Compile and upload without a board attached using keyboard (command-U)
3. Attempt to navigate to error pop-up. Pop-up will disappear before you get to it. 

A better solution would be to put this error (and all errors) in the error console, where it can be read along with other status messages.
